azure-pipelines.yml

```
@@ -45,26 +45,254 @@ variables:
+                    # files at nanoFramework.System.Runtime.Serialization folder
+                    echo "##vso[task.setvariable variable=BUILD_LIBRARY;isOutput=true]true"
+
+                    Write-host "##[command] Build Library"
+                }
+            }
+
+- stage: BuildLibrary
+  displayName: 'Build Library'
+  condition: eq(stageDependencies.BuildPrep.RunPreChecks.outputs['CheckChanges.BUILD_LIBRARY'], 'true')
+  dependsOn:
+  - BuildPrep
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 4 but found 2

(indentation)